code/modules/rituals/ritual_types/divine/ritualcircles.dm

```diff
@@ -0,0 +1,290 @@
+							to_chat(user,span_cultsmall("My devotion to the Plague Queen allowing, her servants crawl up from my throat. Come now, father fly..."))
+							loc.visible_message(span_warning("[user] opens their mouth, disgorging a great swarm of flies!"))
+							playsound(loc, 'sound/misc/fliesloop.ogg', 100, FALSE, -1)
+							flylordstriage(src)
+							user.apply_status_effect(/datum/status_effect/debuff/ritesexpended)
+							spawn(120)
+								icon_state = "pestra_chalky"
+
+/obj/structure/ritualcircle/pestra/proc/flylordstriage(src)
+	var/ritualtargets = view(0, loc)
+	for(var/mob/living/carbon/human/target in ritualtargets)
+		to_chat(target,span_userdanger("You feel them crawling into your wounds and pores. Their horrific hum rings through your ears as they do their work!"))
+		target.flash_fullscreen("redflash3")
+		target.emote("agony")
+		target.Stun(200)
+		target.Knockdown(200)
+		to_chat(target, span_userdanger("UNIMAGINABLE PAIN!"))
+		target.apply_status_effect(/datum/status_effect/buff/flylordstriage)
+
+/obj/structure/ritualcircle/dendor
+	name = "Rune of Beasts"
+	desc = "A Holy Rune of Dendor"
+	icon_state = "dendor_chalky"
+	var/bestialrites = list("Rite of the Lesser Wolf")
+
+/obj/structure/ritualcircle/dendor/attack_hand(mob/living/user)
+	if((user.patron?.type) != /datum/patron/divine/dendor)
+		to_chat(user,span_smallred("I don't know the proper rites for this..."))
+		return
+	if(!HAS_TRAIT(user, TRAIT_RITUALIST))
+		to_chat(user,span_smallred("I don't know the proper rites for this..."))
+		return
+	if(user.has_status_effect(/datum/status_effect/debuff/ritesexpended))
+		to_chat(user,span_smallred("I have performed enough rituals for the day... I must rest before communing more."))
+		return
+	var/riteselection = input(user, "Rituals of Beasts", src) as null|anything in bestialrites
+	switch(riteselection) // put ur rite selection here
+		if("Rite of the Lesser Wolf")
+			if(do_after(user, 50))
+				user.say("RRRGH GRRRHHHG GRRRRRHH!!")
+				playsound(loc, 'sound/vo/mobs/vw/idle (1).ogg', 100, FALSE, -1)
+				if(do_after(user, 50))
+					user.say("GRRRR GRRRRHHHH!!")
+					playsound(loc, 'sound/vo/mobs/vw/idle (4).ogg', 100, FALSE, -1)
+					if(do_after(user, 50))
+						loc.visible_message(span_warning("[user] snaps and snarls at the rune. Drool runs down their lip..."))
+						playsound(loc, 'sound/vo/mobs/vw/bark (1).ogg', 100, FALSE, -1)
+						if(do_after(user, 30))
+							icon_state = "dendor_active"
+							loc.visible_message(span_warning("[user] snaps their head upward, they let out a howl!"))
+							playsound(loc, 'sound/vo/mobs/wwolf/howl (2).ogg', 100, FALSE, -1)
+							lesserwolf(src)
+							user.apply_status_effect(/datum/status_effect/debuff/ritesexpended)
+							spawn(120)
+								icon_state = "dendor_chalky"
+
+/obj/structure/ritualcircle/dendor/proc/lesserwolf(src)
+	var/ritualtargets = view(1, loc)
+	for(var/mob/living/carbon/human/target in ritualtargets)
```

Comment:
once more. i'm not gonna go through and point out every case where this pattern is done but they do all need to be fixed.